generics/middleware/authenticator.js

```diff
@@ -417,8 +509,67 @@ module.exports = async function (req, res, next) {
       }
     }
   }
+
+
+    if (performInternalAccessTokenCheck) {
+      // Validate the presence of required headers
+      let adminOrgId = req.get('orgId');
```

Comment:
@borkarsaish65 why this iis called admin orgId use genaralised variable names that makes sense. if the request coming from tenant admin will this make sense?

Reply:
I have fixed this